desktop/auth.go

```diff
@@ -128,10 +128,6 @@ func logout() *C.char {
 	}
 
 	clearLocalUserData()
-	// Create new user
```

Comment:
We need to create a new user when the user logout, Any reason why we are removing it?

Reply:
It would be great to move a lot of this logic out of the FFI methods to make them more lightweight and maintainable. Having a leaner FFI interface would also simplify the front-end integration and make it easier to switch between different core proxy libraries.. That said, I mainly overlooked adding these back!